agent/ecscni/plugin.go

```diff
@@ -34,9 +36,11 @@ import (
 const (
 	currentCNISpec = "0.3.1"
 	// ECSCNIVersion, ECSCNIGitHash, VPCCNIGitHash needs to be updated every time CNI plugin is updated
-	currentECSCNIVersion = "2018.10.0"
-	currentECSCNIGitHash = "93f4377604504bff92e7555da73b0cba732a4fbb"
-	currentVPCCNIGitHash = "76973f587f9dfb0b40092a78fb5623c9547bc647"
+	currentECSCNIVersion      = "2018.10.0"
+	currentECSCNIGitHash      = "93f4377604504bff92e7555da73b0cba732a4fbb"
+	currentVPCCNIGitHash      = "8b5e552209b0009aecf3c60568b5a5041c6342c0"
```

Comment:
Non-blocker, but will save you future effort to merge to dev. Can you merge the change from mainline instead of updating here to avoid merge conflict from your branch to dev? 

This is updated from this [commit](https://github.com/aws/amazon-ecs-agent/commit/6526bdfab6a52e490b39b2931c0b665a53ab99f1)

Reply:
Merging this to `eni-trunking` branch, will change this directly in that branch after all the trunking related changes are merged to that branch so that our unit tests keep passing during development. Thanks for the heads up. 😄